diff --git a/node_modules/stylelint-config-tailwindcss/scss.js b/node_modules/stylelint-config-tailwindcss/scss.js
index d81090b..b5b8755 100644
--- a/node_modules/stylelint-config-tailwindcss/scss.js
+++ b/node_modules/stylelint-config-tailwindcss/scss.js
@@ -2,7 +2,7 @@ const { rules } = require('./index');
 
 module.exports = {
   rules: Object.entries(rules).reduce(
-    (rules, [k, v]) => Object.assign({ [`scss/${k}`]: v }, rules),
+    (rules, [k, v]) => Object.assign({ [k === 'import-notation' ? k : `scss/${k}`]: v }, rules),
     rules
   ),
 };
